contribution/lang/ja.json

```diff
@@ -1381,16 +1381,6 @@
  	 	 	 	],
  	 	 	 	"trans": "乱数関数に干渉し、優秀なアイテムのランダムドロップ確率を増やすチップ(伝説 ${legendaryChance}%) and エピック (${epicChance}%)。 伝説的なコンテナのロック解除中（最大${maxStack}回）のアイテムの追加ドロップチャンス（${maxStack}回）を20分間獲得します。 クリスマスシーズン中にミステリーコンテナを入手するための追加のチャンス。",
  	 	 	 	"eng": "A shard that contains advance exploits designed to increase random drop chance of high tier items. Gains additional drop chance of Legendary (${legendaryChance}%) and Epic (${epicChance}%) items when unlocking Legendary containers (stack up to ${maxStack} times) for ${time} minutes. Extra chance to get Mystery Container during Christmas season."
- 	 	 	},
- 	 	 	"descriptionv3": {
- 	 	 	 	"vars": [
- 	 	 	 	 	"epicChance",
- 	 	 	 	 	"legendaryChance",
- 	 	 	 	 	"maxStack",
- 	 	 	 	 	"time"
- 	 	 	 	],
- 	 	 	 	"trans": "高層アイテムのランダムドロップチャンスを増やすように設計された事前の悪用を含む断片。 ${time}分の伝説的なコンテナのロック解除時に、伝説（${legendaryChance}％）と叙事詩（${epicChance}％）の追加ドロップチャンスを獲得します。 *重要：バフがアクティブなときにロック解除プロセスが開始されたときにのみ、BUFFは適用されます。",
- 	 	 	 	"eng": "A shard that contains advance exploits designed to increase random drop chance of high tier items. Gains additional drop chance of Legendary (${legendaryChance}%) and Epic (${epicChance}%) items when unlocking Legendary containers (Not stackable) for ${time} minute. *IMPORTANT: The buff will only apply when the unlocking process is started while the buff is active*."
```

Comment:
It looks like there was a new addition during the revision process, and this is what happened. My apologies.

Reply:
ah okay that needs to be translated later :D